packages/vuetify/src/components/VRating/VRating.ts

```diff
@@ -219,7 +223,10 @@ export default mixins(
       }
 
       return this.$createElement(VIcon, this.setTextColor(this.getColor(props), {
-        attrs: { tabindex: -1 }, // TODO: Add a11y support
+        attrs: {
+          tabindex: -1,
+          'aria-label': this.$vuetify.lang.t(this.iconAriaLabel, i + 1),
+        }, // TODO: Add a11y support
```

Comment:
```suggestion
        },
```

Reply:
@MajesticPotatoe I have made renamed the prop and added it to api-generator locale

* [x]  rename `icon-aria-label` -> `icon-label`
* [x]  add prop description to api-generator locale